### DISCUSSION TITLE
Change themes based on the system theme

### DISCUSSION THREAD

**Discussion by orangecoloured** [kovidgoyal/kitty#8149] [Q&A] [answered]
I read [this](https://sw.kovidgoyal.net/kitty/kittens/themes/#change-color-themes-automatically-when-the-os-switches-between-light-and-dark) guide on themes and I have some questions.

1. I do not see an option to set a theme as dark/light/no-preference while choosing.
2. I manually created theme files with names `dark-theme.auto.conf`, `light-theme.auto.conf` and `no-preference-theme.auto.conf`, but it did nothing. I still don't see the themes change based on the system theme. Even after relaunching kitty.

Are there any props in the `kitty.conf` that I should also use?

I also see people using watchers to switch between themes and bash scripts.

What is the most straightforward way to have kitty using proper themes?

**Comment by kovidgoyal** [comment, marked as answer]
Update kitty then use the themes kitten

**Reply by orangecoloured** [reply]
Ah, Fedora is a bit late with the updates. Will check as soon as it's available.

**Reply by orangecoloured** [reply]
On macOS I have 0.38.0 and it works. Thanks!
